gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -244,7 +243,6 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
             )
             kspTask.commandLineArgumentProviders.addAll(kspExtension.commandLineArgumentProviders)
             kspTask.destination = kspOutputDir
-            kspTask.kspOutputDir.value(kspOutputDir)
```

Comment:
Is this field deprecated?

Reply:
It's actually a duplication to the above line.